.github/workflows/release-pypi.yml

```diff
@@ -0,0 +1,79 @@
+---
+name: Release to PyPI
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_sdist:
+    name: Build source package
+    runs-on: 'ubuntu-latest'
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install/Upgrade Python dependencies
+      run: python -m pip install --upgrade pip wheel
+
+    - name: Build and install Python source package
+      run: |
+        python setup.py sdist
+        python -m pip install dist/*.tar.gz
+
+    - name: Run tests
+      run: |
+        pip install pytest pytest-xdist
+        pytest -n auto test/
+
+    - uses: actions/upload-artifact@v2
+      with:
+        path: dist/*.tar.gz
+
+
+  build_bdist:
+    name: Build binary wheels
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # Windows build fails
+        os: [ubuntu-latest, macos-latest]  #  windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    # setup Python for cibuildwheel
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    # for other architectures, see: https://cibuildwheel.readthedocs.io/en/stable/faq/#emulation
+    - name: Build wheels for CPython
+      uses: pypa/cibuildwheel@v2.1.1
+      env:
+        CIBW_SKIP: 'cp310-* pp*'  # 3.10 and pypy builds are broken
```

Comment:
See for 3.10: https://github.com/dralley/hpy/runs/3286978327?check_suite_focus=true#step:3:1181

The pypy issue is the same as before.

Reply:
... and the same answer an well 😅